PWGCF/FemtoUniverse/Core/FemtoUniverseParticleHisto.h

```diff
@@ -209,62 +211,62 @@ class FemtoUniverseParticleHisto
   /// Called by init both in case of reconstructed data/ Monte Carlo, and for Monte Carlo Truth
   /// \tparam T Data type of the particle
   /// \param part Particle
-  template <o2::aod::femtouniverseMCparticle::MCType mc, typename T>
-  void fillQA_base(T const& part)
+  template <o2::aod::femtouniverseMCparticle::MCType mc, typename T, typename H>
+  void fillQA_base(T const& part, H const& h)
```

Comment:
Nice optimization!
Please use more speakable names. Soon nobody will remember what `h` stands for. For example, `histFolder`.

Reply:
Thanks, will do so.